server/mongodb/models/User.js

```diff
@@ -21,7 +20,41 @@ const UserSchema = new Schema({
   archivedReports: {
     type: [
       {
-        cards: [CardSchema],
+        cards: [
+          {
+            card: {
+              type: Schema.Types.ObjectId,
+              ref: "Card",
+            },
+            imgSelections: {
+              type: [Boolean],
+            },
+            noteSelections: {
+              type: [Boolean],
+            },
+            notes: {
+              type: [
+                {
+                  body: {
+                    type: String,
+                    required: true,
+                  },
+                  userId: {
+                    type: String,
+                    required: true,
+                    default: "63c5be6fa7d3c693fa1d335a", // userId for testNonAdmin account
+                  },
+                  date: {
+                    type: Date,
+                    required: true,
+                    default: Date.now(),
+                  },
+                },
+              ],
+              default: [],
+            },
+          },
+        ],
```

Comment:
whatever idiot implemented report-specific notes (me lol) didn't do the same for archived reports. also, archived reports never saved image selections either! this PR brings archived reports in-line with active reports.